bizao-revenue-publisher/src/main/synapse-config/templates/com.apigate.bizao.revenue.publish.call.Template.xml

```diff
@@ -1,4 +1,84 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <template name="com.apigate.bizao.revenue.publish.call.Template" xmlns="http://ws.apache.org/ns/synapse">
-    <sequence/>
+    <sequence>
+
+        <log level="custom">
+            <property name="DIRECTION" value="REQUEST-IN"/>
+            <property name="APP" value="REVENUE-CALL"/>
+            <property name="TRACE_ID" expression="$ctx:TRANSACTION_ID" scope="default" type="STRING"/>
+            <property name="COUNTRY_CODE" expression="$ctx:COUNTRY"/>
+            <property name="MNO_NAME" expression="$ctx:MNO"/>
+            <property name="ORDER_ID" expression="$ctx:ORDER_ID"/>
+            <property name="AMOUNT" expression="$ctx:AMOUNT"/>
+            <property name="PAYMENT_STATUS" expression="$ctx:PAYMENT_STATUS"/>
+        </log>
+
+        <payloadFactory media-type="json">
+            <format>
+                {
+                "type": "$1",
+                "county_code": "$2",
+                "mno_name": "$3",
+                "status": "$4",
+                "amount": "$5",
+                "order_id": "$6",
+                "currency": "$7",
+                "reference": "$8",
+                "date": "$9",
+                "state": "$10",
+                "intTransaction-id": "$11",
+                "extTransaction-id ": "$12",
+                "channel": "$13",
+                "service_provider": "$14"
+                }
+            </format>
+            <args>
+                <arg evaluator="xml" expression="$ctx:TYPE"/>
+                <arg evaluator="xml" expression="$ctx:COUNTRY"/>
+                <arg evaluator="xml" expression="$ctx:MNO"/>
+                <arg evaluator="xml" expression="$ctx:PAYMENT_STATUS"/>
+                <arg evaluator="xml" expression="$ctx:AMOUNT"/>
+                <arg evaluator="xml" expression="$ctx:ORDER_ID"/>
+                <arg evaluator="xml" expression="$ctx:CURRENCY"/>
+                <arg evaluator="xml" expression="$ctx:REFERENCE"/>
+                <arg evaluator="xml" expression="$ctx:DATE"/>
+                <arg evaluator="xml" expression="$ctx:STATE"/>
+                <arg evaluator="xml" expression="$ctx:INTERNAL_TRANSACTION_ID"/>
+                <arg evaluator="xml" expression="$ctx:OPERATOR_TRANSACTION_ID"/>
+                <arg evaluator="xml" expression="$ctx:CHANNEL"/>
+                <arg evaluator="xml" expression="$ctx:SERVICE_PROVIDER"/>
+            </args>
+        </payloadFactory>
+
+        <!--Get revenue publish URL-->
+        <dblookup>
+            <connection>
+                <pool>
+                    <dsName>jdbc/MOBILE_MONEY_DB</dsName>
+                </pool>
+            </connection>
+            <statement>
+                <sql>
+                    <![CDATA[ SELECT revenue_api_url FROM global_settings WHERE id = 1 ]]></sql>
+                <result column="revenue_api_url" name="REVENUE_PUBLISH_URL"/>
+            </statement>
+        </dblookup>
+
```

Comment:
Please add db changes for new column to global_settings

Reply:
Added DB changes to db_script7.sql file in bizao-mobile-money-API repository 

`ALTER TABLE `mobilemoneydb`.`global_settings`
ADD COLUMN `revenue_publish_api_url` VARCHAR(225) NOT NULL AFTER `payment_resource_location`,
ADD COLUMN `revenue_publish_api_bearer_token` VARCHAR(225) NOT NULL AFTER `revenue_publish_api_url`;`